Sources/NIOHTTP1/HTTPTypes.swift

```diff
@@ -80,24 +80,18 @@ public struct HTTPListHeaderIterator: Sequence, IteratorProtocol {
         return self
     }
     
-    @inline(__always)
     public init(headerName: String,
                 headers: HTTPHeaders) {
-        self.init(headerName: headerName.utf8, headers: headers)
-    }
-    
-    public init(headerName: String.UTF8View,
-                headers: HTTPHeaders) {
         self.headers = headers
-        self.headerName = headerName
+        self.headerName = headerName.utf8
     }
 
 }
 
 extension HTTPHeaders {
-    private static let connectionString = "connection".utf8
-    private static let keepAliveString = "keep-alive".utf8
-    private static let closeString = "close".utf8
+    private static let connectionString = "connection"
+    private static let keepAliveString = "keep-alive"
+    private static let closeString = "close"
```

Comment:
There was no need to change these `private static let`, they are just fine using the `.utf8` view.

Reply:
So how could we use the Iterator on them ??? This is why I made the previous commit like that:

Two initializers, with one only delegating to the other without significant cost (using inline)